WooCommerce/WooCommerceTests/Authentication/AuthenticationManagerTests.swift

```diff
@@ -6,6 +6,9 @@ import Yosemite
 
 /// Test cases for `AuthenticationManager`.
 final class AuthenticationManagerTests: XCTestCase {
+
+    let testSiteURL = "https://test.com"
```

Comment:
DRYed the value while at it.

DRYing literals in unit tests can sometimes backfire because it reduces readability, or rather how expressive the tests are.

In this case, though, the URL is not used in any assertion so there's no loss of clarity.